letsencrypt-apache/letsencrypt_apache/obj.py

```diff
@@ -189,7 +189,7 @@ def conflicts(self, addrs):
                     return True
         return False
 
-    def same_server(self, vhost):
+    def same_server(self, vhost, generic=False):
```

Comment:
We should update the docstring to say what generic does.

While slightly outside the scope of this PR, it'd be nice to update the documentation for all parameters/return values.